django/core/signing.py

```diff
@@ -72,10 +74,16 @@ def base64_hmac(salt, value, key, algorithm='sha1'):
     return b64_encode(salted_hmac(salt, value, key, algorithm=algorithm).digest()).decode()
 
 
+def _cookie_signer_key(key):
+    return b'django.http.cookies' + force_bytes(key)  # SECRET_KEY may be str or bytes.
+
+
 def get_cookie_signer(salt='django.core.signing.get_cookie_signer'):
     Signer = import_string(settings.SIGNING_BACKEND)
-    key = force_bytes(settings.SECRET_KEY)  # SECRET_KEY may be str or bytes.
-    return Signer(b'django.http.cookies' + key, salt=salt)
```

Comment:
While I realize we cannot change that now, do we remember why we added `django.http.cookies` here? The salt alone should make sure that we do not clash with other signatures.

Reply:
> ...do we remember why... 

No we don't 😄 
It was [part of the initial implementation a decade ago](https://github.com/django/django/commit/f60d42846365b2bf2f1c9bc7a3007c303122a20b#diff-855893e869078af0cac503844c1a016958ffa925614d70de68340c3cd52f6e55R87).

Reply:
If the user is able to make the server sign arbitrary data of their choosing, using the same salt and key will yield the same signature. However, if we slightly modify the key, they can't. This makes chosen plaintext attacks less vulnerable.